docs/src/how-to/install/web-app-settings.md

```diff
@@ -45,9 +45,9 @@ webapp:
     FEATURE_ENABLE_MEDIA_EMBEDS: "false"
 ```
 
-## Enable extra entropy
+## Enable extra entropy (only on Windows)
 
-The Wire desktop application uses system-dependent source of random bits as an internal entropy source when generating cryptographic keys. In certain cases it may be desired to enable externally generated entropy derived from mouse movement.
+The Wire desktop application uses system-dependent source of random bits as an internal entropy source when generating cryptographic keys. In certain cases it may be desired to enable externally generated entropy derived from mouse movement. This option only affects only Windows users.
```

Comment:
```suggestion
The Wire desktop application uses system-dependent source of random bits as an internal entropy source when generating cryptographic keys. In certain cases it may be desired to enable externally generated entropy derived from mouse movement. This option only affects Windows users.
```